ci/deploy.groovy

```diff
@@ -0,0 +1,20 @@
+pipeline {
+  agent { node { label 'Node' } }
+  tools { nodejs "10.16.3" }
+
+  stages {
+    stage('Run the deploy script') {
+      steps {
+        sh 'printenv'
+        sh 'bin/deploy.sh'
```

Comment:
when testing the jenkins pipeline, i've noticed that if one of the commands fails in the `.sh` script, jenkins still shows the test as passed... i think any error in the `.sh` script should result in a pipeline failure. need to look into what's causing it to pass.

Reply:
yes that's odd is the nonzero status code getting gobbled up because these are somehow getting wrapped in an "outer script" or something?

Reply:
might be a good idea to see what happens when `exit 1` is placed in the bash script.

Reply:
`exit 1` in bash script properly exits the pipeline. if a command in the bash script exits with error code, we need to check for it and exit. i've added checks for important parts of the script.